src/FileIndexer.ts

```diff
@@ -608,6 +629,25 @@ export class FileIndexer {
 
     return this.checker.getTypeAtLocation(node)
   }
+
+  /**
+   * @param sym ts.Symbol with declarations to process.
+   * @returns an array of unique non-empty SCIP symbol values extracted
+   * from ts.Symbol declarations.
+   */
+  private uniqueDeclarationSymbolValues(sym?: ts.Symbol): string[] {
+    if (!sym?.declarations) {
+      return []
+    }
+
+    return Array.from(
+      new Set(
+        sym.declarations
+          .map(declaration => this.scipSymbol(declaration).value)
+          .filter(Boolean)
+      )
+    )
```

Comment:
Now I'm wondering...is there ever a reasonable situation where this array has length more than 1? IDK if we should log an info message or something to indicate that something unusual is going on? 🤔

If this were scip-ruby, I would have added an assert which would only be triggered in DEBUG builds, not sure if there is an equivalent in JS/TS land.